frontend/src/app/QuickStarts.tsx

```diff
@@ -29,28 +27,61 @@ const qsButton = (
 );
 */
 
-export const QuickStarts = ({ children }) => {
-  // @ts-ignore
-  const allQuickStarts = importAll(require.context('../quickstarts', false, /\.ya?ml$/));
+const QuickStarts: React.FC = ({ children }) => {
   const [activeQuickStartID, setActiveQuickStartID] = useLocalStorage('quickstartId', '');
   const [allQuickStartStates, setAllQuickStartStates] = useLocalStorage('quickstarts', {});
+  const [quickStarts, setQuickStarts] = React.useState<QuickStart[]>([]);
   const { pathname: currentPath } = window.location;
   const quickStartPath = '/quickstarts';
+
+  const updateQuickStarts = React.useCallback(
+    (updatedQuickStarts: any) => {
+      if (JSON.stringify(updatedQuickStarts) !== JSON.stringify(quickStarts)) {
+        setQuickStarts(updatedQuickStarts);
+      }
+    },
+    [quickStarts],
+  );
+
+  React.useEffect(() => {
+    let watchHandle;
+    const watchQuickStarts = () => {
+      fetchQuickStarts().then((response) => {
+        if (response.quickStarts) {
+          updateQuickStarts(response.quickStarts);
+        }
+      });
+      watchHandle = setTimeout(watchQuickStarts, 5000);
+    };
+    watchQuickStarts();
+    return () => {
+      if (watchHandle) {
+        clearTimeout(watchHandle);
+        watchHandle = null;
+      }
+    };
+  }, [updateQuickStarts]);
+
   const valuesForQuickStartContext = useValuesForQuickStartContext({
-    allQuickStarts,
+    allQuickStarts: quickStarts || [],
     activeQuickStartID,
     setActiveQuickStartID,
     allQuickStartStates,
     setAllQuickStartStates,
     footer: {
```

Comment:
you can remove the footer section. What this does it show a `SHow all quickstarts` link on the last step of the drawer, but since we won't have the quickstarts catalog later on it won't make sense to route to it.